pyramid_zipkin/request_helper.py

```diff
@@ -53,14 +36,14 @@ def get_trace_id(request):
     completely random trace id.
 
     :param: current active pyramid request
-    :returns: string representaiton of zipkin trace id
+    :returns: a 64-bit hex string
     """
     if 'X-B3-TraceId' in request.headers:
         return request.headers['X-B3-TraceId']
     elif 'zipkin.trace_id_generator' in request.registry.settings:
         return request.registry.settings['zipkin.trace_id_generator'](request)
```

Comment:
Can we put some validation on the result to ensure it is indeed 64bit hex string ^